src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/MLDsa/MLDsaCngTests.Windows.cs

```diff
@@ -52,10 +69,10 @@ public void ImportPrivateKey_NoExportFlag(MLDsaKeyInfo info)
                 AssertExtensions.SequenceEqual(info.PublicKey, export(mldsa)));
 
             MLDsaTestHelpers.AssertExportMLDsaSecretKey(export =>
-                AssertExtensions.ThrowsContains<CryptographicException>(() => export(mldsa), "The requested operation is not supported"));
+                Assert.Equal(MLDsaTestHelpers.NTE_NOT_SUPPORTED, Assert.ThrowsAny<CryptographicException>(() => export(mldsa)).HResult));
 
             MLDsaTestHelpers.AssertExportMLDsaPrivateSeed(export =>
-                AssertExtensions.ThrowsContains<CryptographicException>(() => export(mldsa), "The requested operation is not supported"));
+                Assert.Equal(MLDsaTestHelpers.NTE_NOT_SUPPORTED, Assert.ThrowsAny<CryptographicException>(() => export(mldsa)).HResult));
 
```

Comment:
Should these be using the new helper?  (And the other lines that changed to look for NTE_NOT_SUPPORTED?